src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,78 @@
 <html>
 	<head>
 		<script>
+			function getMetricsUrl() {
```

Comment:
let's inline all in one function, it is fine that it is computed each time, there are several reasons:
- we are contributing here to global scope, we should avoid name collisions, having one function with very specifc name is fine `onVsCodeWorkbenchError`
- we are calling  `getMetricsUrl` immediately, if it breaks in SH or somewhere else it can break loading or execution of some VS Code, it is rather better to do it only in error handler

Reply:
Updated